migrations/deployment-config.json

```diff
@@ -1,3 +1,3 @@
 {
-    "MULTISIG": "0x775d1377223c9338771CbF955A9a53147219ea4A"
+    "MULTISIG": "0xF8368119Bb1073Cf01B841848725d81b542A4c19"
```

Comment:
change to 0xdaeada3d210d2f45874724beea03c7d4bbd41674

Reply:
https://github.com/ribbon-finance/GammaProtocol-OTC/pull/2/commits/c17315e7a4449b9f8f0708c5be35c43aaeddad58